doc/api/tty.md

```diff
@@ -176,6 +176,34 @@ corresponding to this `WriteStream`. The array is of the type
 `[numColumns, numRows]` where `numColumns` and `numRows` represent the number
 of columns and rows in the corresponding [TTY](tty.html).
 
+### writeStream.hasColors([count][, env])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `count` {integer} The number of colors that are requested (minimum 2).
+  **Default:** 16.
+* `env` {Object} An object containing the environment variables to check.
+  **Default:** `process.env`.
```

Comment:
Just a nit... it's not clear from the documentation how the `env` bit works here. Perhaps an example?

Reply:
There are two examples below but I'll try to improve the wording further.

Reply:
PTAL